src/Login/Login.test.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import Login from './Login.js';
+import { screen, fireEvent, render }
+  from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+describe('Login component', () => {
+  it('should have the correct content when rendered', () => {
+    render(<Login 
+      updateUserId={jest.fn()}
+      updateLoginStatus={jest.fn()}
+      changeView={jest.fn()}
+      error=''
+      updateError={jest.fn()}
+    />)
+
+    const emailInput = screen.getByPlaceholderText('Email address');
+    const passwordInput = screen.getByPlaceholderText('Password');
+    const submitBtn = screen.getByRole('button');
+
+    expect(emailInput).toBeInTheDocument();
+    expect(passwordInput).toBeInTheDocument();
+    expect(submitBtn).toBeInTheDocument(); 
+  });
+
+  //test methods are fired when submit btn is clicked: this.handleLogin
```

Comment:
For these you could break out your POSt functionality into its own file and then spy on it being called on click. For the sad path, you would mock out the implementation of this broken out function with a [rejected value](https://jestjs.io/docs/en/mock-function-api#mockfnmockrejectedvalueoncevalue) and make sure that the correct piece of UI renders on the login component in resposne.